gnowsys-ndf/gnowsys_ndf/ndf/templatetags/ndf_tags.py

```diff
@@ -2696,3 +2709,13 @@ def get_filters_data(gst_name):
 							}
 
 	return filter_dict
+
+@get_execution_time
+@register.assignment_tag
+def get_objectid_name(nodeid):
+ 
+ return (node_collection.find_one({'_id':ObjectId(nodeid)}).name)
```

Comment:
line 2717: not proper indent and also not required blank line between def return